docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -1,240 +1,396 @@
 \documentclass{article}
+\usepackage{enumitem}
+\usepackage{hyperref}
 
-\usepackage{booktabs}
-\usepackage{tabularx}
-
-\title{Development Plan\\\progname}
-
-\author{\authname}
-
+\title{Development Plan}
 \date{}
 
-\input{../Comments}
-\input{../Common}
-
 \begin{document}
 
 \maketitle
 
-\begin{table}[hp]
-\caption{Revision History} \label{TblRevisionHistory}
-\begin{tabularx}{\textwidth}{llX}
-\toprule
-\textbf{Date} & \textbf{Developer(s)} & \textbf{Change}\\
-\midrule
-Date1 & Name(s) & Description of changes\\
-Date2 & Name(s) & Description of changes\\
-... & ... & ...\\
-\bottomrule
-\end{tabularx}
-\end{table}
-
-\newpage{}
-
-\wss{Put your introductory blurb here.  Often the blurb is a brief roadmap of
-what is contained in the report.}
-
-\wss{Additional information on the development plan can be found in the
-\href{https://gitlab.cas.mcmaster.ca/courses/capstone/-/blob/main/Lectures/L02b_POCAndDevPlan/POCAndDevPlan.pdf?ref_type=heads}
-{lecture slides}.}
-
```

Comment:
add these back D:

Reply:
@fondsonlu choobydooki what happened DDDDDD; stop nuking our intros

Reply:
added this back